airbyte-workers/src/main/java/io/airbyte/workers/process/KubePodProcess.java

```diff
@@ -345,7 +305,7 @@ public KubePodProcess(ApiClient officialClient,
     Container main = getMain(
         image,
         usesStdin,
-        entrypoint,
+        entrypointOverride,
```

Comment:
now that we no longer need to specify entrypoint - what is the override for? looks like mostly tests?

Reply:
This is used by custom operators.